packages/runtime/container-runtime/src/deltaManagerProxies.ts

```diff
@@ -194,3 +195,19 @@ export class DeltaManagerSummarizerProxy extends BaseDeltaManagerProxy {
 		this.isSummarizerClient = this.deltaManager.clientDetails.type === summarizerClientType;
 	}
 }
+
+export class DeltaManagerPendingOpsProxy extends BaseDeltaManagerProxy {
+	public get minimumSequenceNumber(): number {
+		return (
+			this.pendingStateManager.minimumPendingMessageSequenceNumber ??
+			this.deltaManager.minimumSequenceNumber
+		);
+	}
+
+	constructor(
+		protected readonly deltaManager: IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>,
+		private readonly pendingStateManager: PendingStateManager,
```

Comment:
Can it take a dependency on smaller type? I.e. Pick<PendingStateManager, "minimumPendingMessageSequenceNumber "> ?

Reply:
Updated